entity-service-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_key.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.entity.data.service.v1;
+
+message EntityChangeEventKey {
```

Comment:
Can we make this a separate module (with its own package)? Regular API consumers need not get these protos.